hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledFutureProxy.java

```diff
@@ -63,7 +63,7 @@
 
     private ScheduledTaskHandler handler;
 
-    public ClientScheduledFutureProxy(ScheduledTaskHandler handler, ClientContext context) {
+    ClientScheduledFutureProxy(ScheduledTaskHandler handler, ClientContext context) {
```

Comment:
So the `ClientScheduledFutureProxy` does not offer this kind of best-effort mechanism for notifying of partition and member loss?

Reply:
Never did. I originally added this to server side only with plans to add it to client too. However, after seeing that this might have been a bad idea, i don't want to support it there too. Lets see if this fix proves to be solid in production, and we might reconsider this decision.